plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -291,7 +291,7 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 		},
 	})
 	addControllerRole(&controllerRoles, &controllerRoleBindings, rbacv1.ClusterRole{
-		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "service-controller"},
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "loadbalancer-controller"},
```

Comment:
I'm not super familiar with auth bootstrap but this seems fine in terms of compatability. Would be good if someone from SIG Auth can comment on if there needs to be some clean up code to delete the `service-controller` role.

Reply:
maybe @liggitt or @deads2k :)

Reply:
we do not delete previous default roles on upgrade... it makes upgrade/rollback unsafe. I do question whether the externally visible parts of this change are worth it

Reply:
There may also be CPs who use the add on manager or private IAM to create the service accounts/cluster roles appropriate to their cloud. As such you will be breaking them. This requires significant release notes.

Reply:
@cheftako  We might add an additional `loadbalancer-controller` role instead of modify the existing `service-controller` role. That would not break the compatibility.